### PR TITLE
Default Log_Msg_* dtors

### PR DESCRIPTION
**Pull request by likema**

### CODE REVIEW

File: ACE/ace/Log_Msg_Backend.cpp [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 ACE_BEGIN_VERSIONED_NAMESPACE_DECL
 
-ACE_Log_Msg_Backend::~ACE_Log_Msg_Backend ()
-{
-}
+ACE_Log_Msg_Backend::~ACE_Log_Msg_Backend () = default;
```
Comment (jwillemsen) [inline ACE/ace/Log_Msg_Backend.cpp:5]: Can be done in header

File: ACE/ace/Log_Msg_Android_Logcat.h [hunk 2]
```diff
@@ -35,7 +35,6 @@ ACE_BEGIN_VERSIONED_NAMESPACE_DECL
 class ACE_Export ACE_Log_Msg_Android_Logcat : public ACE_Log_Msg_Backend
 {
 public:
-  ACE_Log_Msg_Android_Logcat () = default;
```
Comment (jwillemsen) [inline ACE/ace/Log_Msg_Android_Logcat.h:38]: Why remove it, when marked with default it is clear that the default constructor is there but is just the default compiler generated

File: ACE/ace/Log_Msg_Callback.cpp [hunk 3]
```diff
@@ -2,8 +2,6 @@
 
 ACE_BEGIN_VERSIONED_NAMESPACE_DECL
 
-ACE_Log_Msg_Callback::~ACE_Log_Msg_Callback ()
-{
-}
+ACE_Log_Msg_Callback::~ACE_Log_Msg_Callback () = default;
```
Comment (jwillemsen) [inline ACE/ace/Log_Msg_Callback.cpp:5]: in header

**Review by jwillemsen — CHANGES_REQUESTED** [review]

File: ACE/ace/Log_Msg_Backend.cpp (hunk 1 above)
Comment (likema) [reply ACE/ace/Log_Msg_Backend.cpp:5]: OK

File: ACE/ace/Log_Msg_Callback.cpp (hunk 3 above)
Comment (likema) [reply ACE/ace/Log_Msg_Callback.cpp]: OK

File: ACE/ace/Log_Msg_Android_Logcat.h (hunk 2 above)
Comment (likema) [reply ACE/ace/Log_Msg_Android_Logcat.h:38]: No matter whether to define the default ctor, the compiler will generate it.

**Comment by jwillemsen** [comment]
Don’t force push braches as it results in more time to review

File: ACE/ace/Log_Msg_Android_Logcat.h (hunk 2 above)
Comment (jwillemsen) [reply ACE/ace/Log_Msg_Android_Logcat.h:38]: Just let it stay, makes clear the compiler is responsible for the default

File: ACE/ace/Log_Msg_Backend.cpp
Comment (jwillemsen) [inline ACE/ace/Log_Msg_Backend.cpp:1]: Looks it can be removed

**Comment by likema** [comment]
> Don’t force push braches as it results in more time to review

Sorry, I just wanted to make the commit(s) in this PR as less as possible

File: ACE/ace/Log_Msg_Backend.cpp
Comment (likema) [reply ACE/ace/Log_Msg_Backend.cpp:1]: OK

File: ACE/ace/Log_Msg_Android_Logcat.h (hunk 2 above)
Comment (likema) [reply ACE/ace/Log_Msg_Android_Logcat.h]: OK

**Review by jwillemsen — APPROVED** [review]

**Pull request merged**